### DISCUSSION TITLE
The possibility that machine learning can work in the mobile phone by this framework.

### DISCUSSION THREAD

**Discussion by iddbh** [beeware/briefcase#1121] [Q&A]
Can the packages like PyTorch work  in the mobile phone by this framework?

**Comment by mhsmith** [comment]
Yes, we currently have the following ML packages available for Android:
* scikit-learn 0.24.1
* tensorflow 2.1.0
* tflite-runtime 2.5.0
* torch 1.8.1

Unfortunately none of them are available for iOS yet.

**Reply by iddbh** [reply]
Thank you for your advice.
